test/unit/modules/helpers/get-n-parents-spec.js

```diff
@@ -136,6 +152,36 @@ describe('RecordingLibrary.helpers.getNParents', () => {
         type: null,
         nth_parent: 1,
       },
+      {
+        class: null,
+        id: null,
+        name: null,
+        all_attrs: {
+          unique_selector: 'html',
+        },
+        href: null,
+        node_name: 'HTML',
+        tag_name: 'HTML',
+        text: null,
+        title: null,
+        type: null,
+        nth_parent: 2,
+      },
+      {
+        class: null,
+        id: null,
+        name: null,
+        all_attrs: {
+          unique_selector: null,
+        },
+        href: null,
+        node_name: '#document',
+        tag_name: undefined,
+        text: null,
+        title: null,
+        type: null,
+        nth_parent: 3,
+      },
```

Comment:
This block captures the parent nodes that weren't being captured before, given the test node was not actually a part of any DOM. I added them here because they are expected.

In other test files, since we're not actually focused on the node parents, I used a more general "any array" matcher to avoid unnecessarily verbose & complex specs.